src/nvim/drawline.c

```diff
@@ -287,7 +287,9 @@ static void draw_virt_text(win_T *wp, buf_T *buf, int col_off, int *end_col, int
       col = draw_virt_text_item(buf, item->draw_col, vt->data.virt_text,
                                 vt->hl_mode, max_col, vcol);
     }
-    item->draw_col = INT_MIN;  // deactivate
+    if (!vt || !(vt->flags & kVTRepeatLinebreak) || item->kind == kDecorKindUIWatched) {
```

Comment:
The last condition is again never reaches because `vt` is only assigned when `item->kind == kDecorKindVirtText`, so PVS will probably give a warning here.

Reply:
Oh right, your first review comment I took your first review comment to mean a change was needed.